pkg/registry/apis/identity/legacy/query_display.sql

```diff
@@ -0,0 +1,13 @@
+SELECT org_user.org_id, u.id, u.uid, u.login, u.email, u.name, 
+  u.created, u.updated, u.is_service_account, u.is_disabled, u.is_admin
+  FROM {{ .Ident "user" }} as u JOIN org_user ON u.id = org_user.user_id
+ WHERE org_user.org_id = {{ .Arg .Query.OrgID }} AND ( 1=2
```

Comment:
`1=2` to not return anything if no uids or ids is passed?

Reply:
correct -- this is really a `false OR (match uids) OR (match ids)` the templating gets too gnarly when you have to check if the previous exists or not